src/utils/Terminal.php

```diff
@@ -191,12 +221,10 @@ public static function isInit() : bool{
 	public static function toANSI(string $string) : string{
 		$newString = "";
 		foreach(TextFormat::tokenize($string) as $token){
-			$newString .= match($token){
+			$newString .= match ($token) {
```

Comment:
This is probably was done by accident

Reply:
Yeah, but the PhpStorm formatter does this and there doesn't seem to be a way to stop it. I've stopped trying to fight with it at this point.

I guess this is the price we pay for maintaining a non-standard code style...